README.md

```diff
@@ -1,63 +1,101 @@
+6. Copy the `current_key` and `mobilesdk_app_id` from the json file you just downloaded into the android object in the `FirebasePlatformOverrides` variable in `./config/app.js`. Then copy the `client_id` on line 18 to line 47 of the same `app.js` file.
+7. Again from the downloaded json file, copy the `current_key` and the `certificate_hash` to lines 54 and 55 of `./mobile/app.json`.
+8. Create `.runtimeconfig.json` in `./server/functions` with the following:
+
+	```
+	{
+	  "envs": {
+	    "project_name": "[appName]",
+	    "sendgrid_api_key": "...",
+	    "sendgrid_emails_validation_api_key": "...",
+	    "is_prod": "false", // set false for staging,
+	    "devlogin": true,
+	    "twilio_phone_number_from": "...",
+	    "twilio_auth_token": "...",
+	    "twilio_account_sid": "..."
+	  }
+	}
+	```
+9. Add the Sendgrid API key to the `.runtimeconfig.json` file you just created (get this key from a dev on the team).
+10. Now create `.env` in `./server/functions` with the following:
+
+	```
+	GOOGLE_APPLICATION_CREDENTIALS=/path/to/json/credentials
+	```
+	(we will fill in the path in the next step)
+11. Go to the Google Cloud Platform [here](https://console.cloud.google.com/iam-admin/serviceaccounts?project=bipolarbridges). On the "App Engine default service account", click the three dots, select "Create key", and download the json file. Fill in the path to this downloaded file in the `.env` from the previous step.
+12. Open the `./mobile/ios` folder in Xcode. Go to the "Info" tab and scroll down to "URL Types". Add a new URL type with a "Role" of "Editor" and in "URL Schemes" paste in the `REVERSED_CLIENT_ID` from the plist file you downloaded earlier.
+
+
+## Run Setup Script
 
 Navigate back to the root directory and run:
+
 ```
-yarn all
+./bin/setup.bash
 ```
 
-### Configuration files references
+## Running the App
+
+### Run & build app
 
-Environment config file `./server/functions/.env` (should be created if not exists).
-    * .env file content `GOOGLE_APPLICATION_CREDENTIALS=/path/to/json/credentials`
-   
-### Validate fields in app.json
+* **IOS**: To run project locally run `yarn ios` from the `./mobile` directory
+* **Android**: To run project locally the first time, open the android directory in android studio, sync with gradle and run the app. In subsequent runs, run `yarn android` from the `./mobile` directory
 
-1. ios.bundleIdentifier should be ```"com.maslo.evolution"```
-2. ios.config.googleSignIn.reservedClientId should be ```com.googleusercontent.apps.858064288439-02l37uab6frvq67228g5taua7c112pii```
-3. android.package should be ```"com.maslo.evolution"```
-4. android.config.googleSignIn.apiKey should be ```AIzaSyBbEpbXQY-xKsrMzrsdXNBuBdHzsTpPMBw```
-5. android.config.googleSignIn.certificateHash should be ```858064288439-oq1lit38a9cc39sb6dhcrrjmmk8k8i6t.apps.googleusercontent.com```
+## Maslo Dashboard
 
+The staging dashboard is currently hosted [here](https://bipolarbridges.web.app/)
```

Comment:
Usually, we wouldn't put specifics like a URL to an instance of this app in the README since others who use this repo may install it in a different location. Does our set up end up creating the app in this URL? If not, then this shouldn't be in here technically.

Reply:
Our setup steps do create the app at this URL. It's currently already deployed there and anytime a dev changes the dashboard and redeploys it, it will re-create the dashboard at this url. The url is from our firebase project, so it won't vary per instance of the app on each devs local computer if they follow the setup steps, but if another company were to use this repo with their own firebase project, it would be different (if that's what you mean?). Do you still want it taken out?